src/generator/comparator/DefaultConstantComparator.php

```diff
@@ -12,19 +18,25 @@
  * Orders them by lower cased first, then upper cased
  */
 class DefaultConstantComparator implements Comparator {
-
-	private $comparator;
+	private DefaultUseStatementComparator $comparator;
 
 	public function __construct() {
 		$this->comparator = new DefaultUseStatementComparator();
 	}
 
 	/**
-	 * @param PhpConstant $a
-	 * @param PhpConstant $b
+	 * @param mixed $a
+	 * @param mixed $b
+	 *
+	 * @throws \TypeError
+	 *
+	 * @return int
 	 */
-	public function compare($a, $b) {
+	public function compare(mixed $a, mixed $b): int {
```

Comment:
Ah shooot. I was suggesting this:

```suggestion
	public function compare(PhpConstant $a, PhpConstant $b): int {
```

until I realized without `mixed` it wouldn't match the `Comparator` interface. Dang php, it's time for generics 😂